deluge/tests/test_tracker_icons.py

```diff
@@ -24,8 +22,8 @@
 @pytest.mark.internet
 class TrackerIconsTestCase(BaseTestCase):
 
-    if windows_check():
-        skip = 'cannot use os.path.samefile to compair on windows(unix only)'
+    # if windows_check():
+    #     skip = 'cannot use os.path.samefile to compair on windows(unix only)'
```

Comment:
We should always aim to delete instead of commenting out code

Reply:
forgot I commented out this code.. whoops